jobs/env/ci-kubernetes-e2e-scalability.env

```diff
@@ -1,19 +1,19 @@
 ### job-env
 
+### Common env variables for all scalability-related suites.
+
 # Override GCE defaults.
 NODE_SIZE=n1-standard-1
 NODE_DISK_SIZE=50GB
-NUM_NODES=100
-ALLOWED_NOTREADY_NODES=1
 REGISTER_MASTER=true
+LOGROTATE_MAX_SIZE=5G
+
+# Use IP-aliases for scalability tests.
 KUBE_GCE_ENABLE_IP_ALIASES=true
 
 # Switch off image puller to workaround #44701
 PREPULL_E2E_IMAGES=false
-# Reduce logs verbosity
-TEST_CLUSTER_LOG_LEVEL=--v=2
```

Comment:
Why are we not setting `--v=2` in the common env file?

Reply:
On purpose - different jobs set different verbosity.
Once our configs are clearer, we may think more how to fix that, but for now I don't want to mix that.

Reply:
ok, sgtm